CMakeLists.txt

```
@@ -571,7 +571,7 @@ include_directories(SYSTEM velox/external)
 
 # these were previously vendored in third-party/
 if(NOT VELOX_DISABLE_GOOGLETEST)
-  set(GTest_SOURCE AUTO)
+  set_source(GTest)
   resolve_dependency(GTest)
```

Actually, if we have a minimum version I wonder why it's not set here @majetideepak @kgpai ? https://github.com/facebookincubator/velox/blob/main/CMake/resolve_dependency_modules/README.md doesn't seem to list it though
```suggestion
  resolve_dependency(GTest 1.13)
```

---

We have no minimum versions for gtest and glog. We depend on the default from the platform/system package manager.

---

I specified minmimun versionf for gtest and glog.

---

> to successully build we need v1.13.0

What issue do you have? I just checked and our CI containers use 1.11 so I don't think that minimum version is required for velox. 

So in that case please remove the minimum version again unless you can demonstrate that they are needed for current `main` state (if you need it for a PR please handle it there)

---

I just rollback the minimum versions and I will try to reproduce it (for the environment has changed needs some time)